app/javascript/widget/store/modules/conversation.js

```diff
@@ -61,15 +61,22 @@ export const getters = {
     return Object.keys(conversationGroupedByDate).map(date => {
       const messages = conversationGroupedByDate[date].map((message, index) => {
         let showAvatar = false;
-        if (index === conversationGroupedByDate[date].length - 1) {
+        if (
+          message.content_type === 'form' &&
```

Comment:
Make this check a variable for better readability ?

Reply:
Fixed.